centreon/packages/ui/src/Graph/Chart/Chart.cypress.spec.tsx

```
@@ -142,13 +144,13 @@
     .and('equal', '376.203125');
 
   cy.findByTestId('graph-interaction-zone').then((graph) => {
-    expect(Number(graph[0].attributes.width.value)).to.be.greaterThan(1170);
+    expect(Number(graph[0].attributes.width.value)).to.be.lessThan(1220);
   });
 };
 
 describe('Line chart', () => {
   describe('Tooltip', () => {
-    it('displays a tooltip when the graph is hovered', () => {
+    it.only('displays a tooltip when the graph is hovered', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)